internal/nostr/nostr_test.go

```
@@ -4,346 +4,151 @@ import (
 
-	err = req.Sign(privateKey)
+	payload, err := json.Marshal(body)
 	if err != nil {
-		return &nostr.Event{}
+		t.Fatalf("marshal request body: %v", err)
 	}
 
-	return req
-}
+	e := echo.New()
+	req := httptest.NewRequest(method, target, bytes.NewBuffer(payload))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)